Streetcode/Streetcode.XIntegrationTest/ControllerTests/Streetcode/Create/StreetcodeCreateControllerTests.cs

```diff
@@ -0,0 +1,97 @@
+           : base(factory, "/api/Streetcode")
+        {
+
+        }
+
+        [Fact]
+        [ExtractCreateTestStreetcode]
+        public async Task Create_ReturnsSuccessStatusCode()
+        {
+            // Arrange
+            var streetcodeCreateDTO = ExtractCreateTestStreetcode.StreetcodeForTest;
+
+            // Act
+            var response = await client.CreateAsync(streetcodeCreateDTO);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+        }
+
+
+        [Fact]
+        [ExtractCreateTestStreetcode]
+        public async Task Create_CreatesNewStreetcode()
+        {
+            // Arrange
+            var streetcodeCreateDTO = ExtractCreateTestStreetcode.StreetcodeForTest;
+
+            // Act
+            var response = await client.CreateAsync(streetcodeCreateDTO);
+            var streetcodeId = StreetcodeIndexFetch.GetStreetcodeByIndex(streetcodeCreateDTO.Index);
+            var getResponse = await client.GetByIdAsync(streetcodeId);
+            var fetchedStreetcode = CaseIsensitiveJsonDeserializer.Deserialize<StreetcodeContent>(getResponse.Content);
+
+            // Assert
+            // Check if the fetched Streetcode matches the one we created
+            Assert.Equal(streetcodeCreateDTO.Title, fetchedStreetcode.Title);
+            Assert.Equal(streetcodeCreateDTO.TransliterationUrl, fetchedStreetcode.TransliterationUrl);
+        }
+
+        [Fact]
+        [ExtractCreateTestStreetcode]
+        public async Task Create_WithInvalidData_ReturnsBadRequest()
+        {
+            // Arrange
+            var streetcodeCreateDTO = ExtractCreateTestStreetcode.StreetcodeForTest;
+            streetcodeCreateDTO.Title = null;  // Invalid data
+
+            // Act
+            var response = await client.CreateAsync(streetcodeCreateDTO);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
+        }
+
+        [Fact]
+        [ExtractTestStreetcode]
+        [ExtractCreateTestStreetcode]
+        public async Task Create_WithExistingStreetcode_ReturnsConflict()
+        {
+            // Arrange
+            var streetcodeCreateDTO = ExtractCreateTestStreetcode.StreetcodeForTest;
+            streetcodeCreateDTO.Index = ExtractTestStreetcode.StreetcodeForTest.Index;
+
+            // Act
+            var response = await client.CreateAsync(streetcodeCreateDTO);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
+        }
+
+        [Fact]
+        [ExtractCreateTestStreetcode]
+        public async Task Create_WithLongTransliterationUrl_ReturnsBadRequest()
+        {
+            // Arrange
+            var streetcodeCreateDTO = ExtractCreateTestStreetcode.StreetcodeForTest;
+            streetcodeCreateDTO.TransliterationUrl = new string('a', 151);  // TransliterationUrl length exceeds the maximum limit
```

Comment:
please create a const with name of `TransliterationUrlMaxLength = 150` and then do it like
`streetcodeCreateDTO.TransliterationUrl = new string('a', TransliterationUrlMaxLength + 1);`

It would make the comment not necessary

Reply:
Done